notifications.bs

```diff
@@ -734,9 +734,14 @@ constructor steps are:
 
 <h3 id=static-members>Static members</h3>
 
-<p>The static <dfn attribute for=Notification><code>permission</code></dfn> getter steps are to
-return the <a>permission</a> for the <a>current settings object</a>'s
-<a for="environment settings object">origin</a>.
+<p>The static <dfn attribute for=Notification><code>permission</code></dfn> getter steps are:
+
+<ol>
+ <li><p>Let <var>settings</var> be the <a>current settings object</a>.
+
+ <li><p>If <var>settings</var> is a <a>secure context</a>, then return the <a>permission</a> for
+ <var>settings</var>'s <a for="environment settings object">origin</a>. Otherwise, return
+ "<code>denied</code>".
```

Comment:
This is weird as actually we should run the permission query algorithm here too, no? And change prompt to default? And remove section that defines `<a>permission</a>` inline.

Reply:
OK, let's see if we can make a common <dfn> for both to use.